framework/include/base/MooseError.h

```diff
@@ -140,4 +140,23 @@
 
 #define mooseCheckMPIErr(err) do { if (err != MPI_SUCCESS) { if (libMesh::global_n_processors() == 1) print_trace(); libmesh_here(); MPI_Abort(libMesh::GLOBAL_COMM_WORLD,1); exit(1); } } while (0)
 
+#define mooseInfo(msg)                                                  \
+  do                                                                    \
+  {                                                                     \
+    mooseDoOnce(                                                        \
+      {                                                                 \
+        std::ostringstream _info_oss_;                                  \
+                                                                        \
+        _info_oss_                                                      \
+          << (Moose::_color_console ? XTERM_BLUE : "")                  \
+          << "\n\n*** Info ***\n"                                       \
+          << msg                                                        \
+          << "\nat " << __FILE__ << ", line " << __LINE__               \
+          << (Moose::_color_console ? XTERM_DEFAULT : "")               \
+          << "\n\n";                                                    \
+        Moose::err << _info_oss_.str() << std::flush;                   \
```

Comment:
Do we really want "info" going to the error stream? I think it might be better suited on the `out` stream.